src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaSendMessageRequest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda;
+
+import java.util.Map;
+
+public class CamundaSendMessageRequest {
```

Comment:
Is this used in Task Management as it sounds starting a workflow?  
Maybe for this is for testing purposes?

Reply:
Only used it for testing purposes as on the functional tests I need to setup a task, I guess this is one of those objects that we will copy and paste in the other projects that need setting up tasks for functional tests.